.prettierignore

```diff
@@ -10,5 +10,5 @@ data-source
 typechain-types
 public/snippets
 public/search-index.json
-src/pages/chainlink-nodes/v1/running-a-chainlink-node.mdx
-vercel.json
\ No newline at end of file
+src/content/chainlink-nodes/v1/running-a-chainlink-node.mdx
```

Comment:
Could you create an issue for this? Arbitrarily adding files to .prettierignore is a bad practice and should be avoided. I'm sure you can solve this rather trivially.